<?xml version="1.0"?>

<launch>

  <node name="inference_services" pkg="src2_object_detection" type="inference_service.py" output="screen">
  </node>

  <node name="obstacle_detection" pkg="src2_object_detection" type="obstacle_detection_multiple_rovers.py" output="screen">
  </node>

  <group ns="small_scout_1">
    <node name="object_estimation_service" pkg="src2_object_detection" type="object_estimation_service.py" output="screen">
    </node>
  </group>

  <group ns="small_scout_2">
    <node name="object_estimation_service" pkg="src2_object_detection" type="object_estimation_service.py" output="screen">
    </node>
  </group>

</launch>
